example/view/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+set(CMAKE_FOLDER "${CMAKE_FOLDER}/algorithm")
```

Comment:
I'm a CMake n00b. What does this do?

Reply:
Appends `/algorithm` to `${CMAKE_FOLDER}`.